### DISCUSSION TITLE
How do I obtain the domain name of the current site from boot/axios.js

### DISCUSSION THREAD

**Discussion by gh81997167** [quasarframework/quasar#13392] [CLI - SSR mode]
Our request is based on the website's own domain name to request the interface, for example, the corresponding API request of a.com is a.com/api, the corresponding API request of b.com is b.com/api,  Found unavailable windos.localhost.host and process.env.api_url in boot/axios.js  
Process.env. API_URL can be used in the development environment but not in the production environment  
Is there any way to configure automatic domain name acquisition for the current site in boot/axios.js

**Comment by yusufkandemir** [comment]
You can still use `process.env.API_URL` by correctly passing it when building your app(`quasar build`), if you want to use the same build in multiple addresses, this won't work. To get the active hostname(e.g. `a.com`), you can use [`window.location.hostname`](https://developer.mozilla.org/en-US/docs/Web/API/Location/hostname). This won't work on SSR when running on the server, so you might need to check if running on the server, e.g. `if(process.env.SERVER)` and use [`ssrContext.req.hostname`](https://quasar.dev/quasar-cli-vite/developing-ssr/ssr-context#anatomy-of-ssrcontext) in that case.
